reference/security-compliance/faq.md

```diff
@@ -0,0 +1,114 @@
+---
+title: FAQ
+description: Frequently asked questions regarding Security & Compliance
+---
+
+Detailed below are the answers to some frequently asked questions about PlaceOS, data storage and security.
+
+## Data
+
+**Q: How does PlaceOS manage data, protecting customer data and governs customer data?**
+
+A: Most data is not stored internally within the system. 
+
+Sources of truth such as Office365 hold the data.  
+
+Manipulated data is stored ephemerally in memory and access to any data requires authentication, authorisation to access is determined by the defined processes.
+
+**Q: Can you please outline each type of datastore / database / data repository included in your system?**
+
+A: The PlaceOS Solution uses Couchbase as the primary database and datastore. 
```

Comment:
RethinkDB - there are a few references to couchbase here

Reply:
updated all references to rethink.